.tekton/collector-pull-request.yaml

```diff
@@ -0,0 +1,391 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
```

Comment:
[not about this line]

Regarding

> We need to follow up on building the driver-builds in Konflux as well eventually (<https://issues.redhat.com/browse/ROX-22244>).

in the PR description.

I meant ROX-22244 not for onboarding driver builds to Konflux. ROX-22244 was about a different problem. I expanded the ticket's description providing more context. Please have a look and let's follow discussions with the Collector team to jointly discover their plans and be in sync how this could match ACS Konflux onboarding.

Reply:
updated PR description. The additional context makes it clear what ROX-22244 should achieve and we can start with this after MVP of MVP